widget/src/apiCaller.js

```diff
@@ -0,0 +1,20 @@
+// Options:
+// local
+// staging - https://staging-api.infinite.industries/events/current/verified/
+// production - https://api.infinite.industries/events/current/verified/
+
+// Create methods ^%$#^%$#!!!!
+
+
+export function APICaller(callback){
```

Comment:
If this used `export default function`... instead then it could be imported without the extra brackets, e.g. `import APICaller from './apiCaller.js'` instead of `import { APICaller }`...

Reply:
Done.